SpywareFilter/sections/general_url.txt

```diff
@@ -113,6 +113,7 @@ _analytics.js?
 .com/b.gif?
 .jp/b.gif?
 /v.gif?t=
+/magazine-newspaper-pro/analytics.js
```

Comment:
Why did you make a general rule?
Also this request has a 404 status and it contains nothing.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/118104815-4bc43f80-b3e4-11eb-9857-480adeb624d9.png)

</details><br/>